frontend/package.json

```diff
@@ -191,7 +191,7 @@
     "@types/classnames": "^2.2.11",
     "@types/faker": "^5.5.9",
     "@types/jest": "^29.1.2",
-    "@types/node": "^14.14.10",
+    "@types/node": "18.11.7",
```

Comment:
Does this need the `^` or do we want to lock it in to this specific patch version?

Reply:
Good catch, when I installed it it didn't had the ^.